packages/buttons/src/elements/Button.tsx

```diff
@@ -13,28 +13,53 @@ import { useSplitButtonContext } from '../utils/useSplitButtonContext';
 import { StartIcon } from './components/StartIcon';
 import { EndIcon } from './components/EndIcon';
 
-const ButtonComponent = forwardRef<HTMLButtonElement, IButtonProps>((props, ref) => {
-  const splitButtonContext = useSplitButtonContext();
+const ButtonComponent = forwardRef<HTMLButtonElement, IButtonProps>(
```

Comment:
`forms` stories rely on `buttons`, thus changes to `buttons` transient props were necessary.